model_hub/model_hub/mmdetection/_callbacks.py

```diff
@@ -140,7 +140,10 @@ def __init__(self, context: det_torch.PyTorchTrialContext, model: torch.nn.Modul
         self.model = model
 
     def on_validation_start(self) -> None:
-        if self.context.distributed.get_size() > 1:
+        if (
+            self.context.distributed.get_size() > 1
+            and self.context._distributed_backend.use_horovod()
```

Comment:
let's assert that backend is use_torch and then switch the broadcast statements below to those from the torch.distributed library

Reply:
discussed, mmdetection disables torch DDP's `broadcast_buffers` for some reason, which handles this.

Reply:
discussed, mmdetection disables torch DDP's `broadcast_buffers` for some reason, which handles this.